docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -320,14 +321,10 @@ should be executed before this changes.
 **Changes in the `/config/manager` folder:**
 1. Edit `/config/manager/manager.yaml` and remove the `--metrics-bind-addr=127.0.0.1:8080` arg from the `args` list.
 
-**Changes in the `/config/rbac` folder:**
```

Comment:
I dropped this because it's already done in `"Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls"`. If we included this here intentionally, it's fine for me to add it again.

Reply:
This was my mistake.
We should remove this from "Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls" and leave them only here (those changes are related to the rbac-proxy removal, not to single manager changes)

Reply:
@fabriziopandini https://github.com/kubernetes-sigs/cluster-api/pull/4797